pyfolio/tears.py

```diff
@@ -347,8 +384,14 @@ def create_bayesian_tear_sheet(returns, benchmark_rets, live_start_date,
         trading, after its backtest period.
     return_fig : boolean, optional
         If True, returns the figure that was plotted on.
+    set_context : boolean, optional
+        If True, set default plotting style context.
     """
 
+    if set_context:
```

Comment:
If this boilerplate is going to be everywhere, we might want to factor it out into a decorator.